<!-- Load ROS params to enable Spot to open a chest of drawers -->
<launch>

    <arg name="open_traj_path" default="/docker/spot/TODO" />
    <param name="/spot/filepaths/open_drawer_traj" value="$(arg open_traj_path)" />

    <param name="/spot/open_drawer/active" value="true" />

</launch>